### PR TITLE
Add supported file extensions for unovonv

### PR DESCRIPTION
**Pull request by remoteexception**
Added the following file extensions for unoconv:
- tif
- jpeg
- odp

### CODE REVIEW

File: pkg/modules/libreoffice/unoconv/unoconv.go [hunk 1]
```diff
@@ -243,9 +244,11 @@ func (mod Unoconv) Extensions() []string {
 		".swf",
 		".sxd",
 		".sxw",
+		".tif",
```
Comment (gulien) [inline pkg/modules/libreoffice/unoconv/unoconv.go:247]: I didn't find `.tif` in the list of supported file extensions (`unoconv --show`) 🤔  `.tif` == `.tiff` ?

Comment (remoteexception) [reply pkg/modules/libreoffice/unoconv/unoconv.go:247]: Yes you are right. But unoconv actually supports .tif.

Comment (remoteexception) [reply pkg/modules/libreoffice/unoconv/unoconv.go:247]: jpeg isn't in the list too, but is supported by unoconv btw.

**Review by gulien — APPROVED** [review]

**Comment by gulien** [comment]
Thanks @remoteexception 😄

**Pull request merged**